marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/mappingWithCustomHeader.cy.tsx

```diff
@@ -137,28 +125,23 @@ describe("Create and verify load steps, map step and flows with a custom header"
   it("Add Map step to new flow and Run", {defaultCommandTimeout: 120000}, () => {
     cy.visit("/tiles/curate");
     cy.waitForAsyncRequest();
-
-    cy.log("**Open Order to see steps**");
-    curatePage.expandAccordian("Order");
+    curatePage.toggleEntityTypeId("Order");
```

Comment:
We use expand in certain places because sometimes the accordion could be closed. (intermittent issues we used to have)
Be careful with this changes.